docker-compose_developer.yml

```diff
@@ -58,6 +73,9 @@ services:
 
   kibana:
     image: kibana:7.14.1
+    profiles:
```

Comment:
same with kibana. useful in every case?

Reply:
I thought that Kibana is just a "nice-to-have" besides the web-app? Is it really necessary for development, especially when we mind the limited resources on dev-machines?